packages/data/platform/users_flutter/lib/src/storage_json.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:convert';
+
+import 'package:json_annotation/json_annotation.dart';
+import 'package:storage_access/storage_access.dart' as storage;
+import 'package:users/users.dart';
+
+part 'storage_json.g.dart';
+
+String _storageFileName = "local_users.json";
+
+@JsonSerializable()
+class UsersStorageJson implements Users {
+  final List<User> users;
```

Comment:
can be private?

Reply:
Can't, but we might make it singleton.
1. it is used to create AuthFlutter.
2. It must be always the same instanse while app is running.
3. It is "provided" in widget tree, just in case.